### PR TITLE
Feature develop update install files

### PR DESCRIPTION
**Pull request by jprestop**
## Expected Differences ##

- [x] Do these changes introduce new tools, command line arguments, or configuration file options? **[No]**</br>
If **yes**, please describe:</br>

- [x] Do these changes modify the structure of existing or add new output data types (e.g. statistic line types or NetCDF variables)? **[No]**</br>
If **yes**, please describe:</br>

## Pull Request Testing ##

- [x] Describe testing already performed for these changes:</br>

Compilation script tested on Hera by John HG.  Modulefiles and config files tested on various machines.

- [x] Recommend testing for the reviewer(s) to perform, including the location of input datasets, and any additional instructions:</br>

Review files if desired

- [x] Do these changes include sufficient documentation updates, ensuring that no errors or warnings exist in the build of the documentation? **[Yes]**

- [x] Do these changes include sufficient testing updates? **[Yes]**

- [x] Will this PR result in changes to the test suite? **[No]**</br>
If **yes**, describe the new output and/or changes to the existing output:</br>

- [x] Please complete this pull request review by **ASAP want to merge before any changes are made to the compilation script for Issue #2514**.</br>

## Pull Request Checklist ##
See the [METplus Workflow](https://metplus.readthedocs.io/en/latest/Contributors_Guide/github_workflow.html) for details.
- [x] Review the source issue metadata (required labels, projects, and milestone). **no issue - regular updates of files as installations for new versions occur**
- [x] Complete the PR definition above.
- [x] Ensure the PR title matches the feature or bugfix branch name.
- [x] Define the PR metadata, as permissions allow.
Select: **Reviewer(s)**
Select: **Organization** level software support **Project** or **Repository** level development cycle **Project**
Select: **Milestone** as the version that will include these changes
- [ ] After submitting the PR, select **Development** issue with the original issue number.
- [ ] After the PR is approved, merge your changes. If permissions do not allow this, request that the reviewer do the merge.
- [ ] Close the linked issue and delete your feature or bugfix branch from GitHub.

### CODE REVIEW

File: internal/scripts/installation/compile_MET_all.sh [hunk 1]
```diff
@@ -430,17 +437,22 @@ if [ $COMPILE_G2CLIB -eq 1 ]; then
   rm -rf ${LIB_DIR}/g2clib/g2clib*
   tar -xf ${TAR_DIR}/g2clib*.tar -C ${LIB_DIR}/g2clib
   cd ${LIB_DIR}/g2clib/g2clib*
-  sed -i 's|INC=.*|INC=-I${LIB_DIR}/include -I${LIB_DIR}/include/jasper|g' makefile
+  # JHG Switch sed commands from using single quotes to double-quotes since
```
Comment (JohnHalleyGotway) [inline internal/scripts/installation/compile_MET_all.sh]: Julie, please remove the comment lines which include `# JHG`. Those are just notes to myself I used at breadcrumbs to remind myself where I'd made changes.

Comment (jprestop) [reply internal/scripts/installation/compile_MET_all.sh]: Thanks @JohnHalleyGotway.  I have removed the JHG comments.

**Review by JohnHalleyGotway — APPROVED** [review]
Julie, I see that all the changes are limited to internal/scripts/installation. As such, they will not affect the logic of the code.

Scrolling quickly through the diffs, I don't see any obvious issues. But I completely defer to you on how these setting work on the target installation machines.

**Comment by JohnHalleyGotway** [comment]
I'm proceeding with merging this PR to include it in the MET 11.1.0-beta2 release.

**Pull request merged**